### PR TITLE
Copyright header check improvements

### PR DESCRIPTION
**Pull request by rafalste**
Make sure to look for all files that need to have copyright header included.

Needs https://github.com/intel-innersource/frameworks.validation.opencas.test-framework/pull/61

### CODE REVIEW

**Comment by rafalste** [comment]
retest this please

**Review by karolinavelkaja — DISMISSED** [review]

File: configure.d/1_bd_part_count.conf [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-. $(dirname $3)/conf_framework
+. $(dirname $3)/conf_framework.sh
```
Comment (arutk) [inline configure.d/1_bd_part_count.conf:7]: I think we should not update the copyright date in a files where the content doesn't change

**Review by arutk — CHANGES_REQUESTED** [review]

File: configure.d/1_bd_part_count.conf (hunk 1 above)
Comment (arutk) [reply configure.d/1_bd_part_count.conf:7]: sorry, this comment is misplaced

File: tools/version2sha.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright(c) 2021 Intel Corporation
+# Copyright(c) 2021-2022 Intel Corporation
```
Comment (arutk) [inline tools/version2sha.sh:3]: I think we should not update the copyright date in a files where the content doesn't change

Comment (rafalste) [reply tools/version2sha.sh]: Fixed with this PR: https://github.com/Open-CAS/open-cas-linux/pull/1340

**Review by arutk — APPROVED** [review]

**Review by robertbaldyga — APPROVED** [review]

**Pull request merged**